src/pymodaq/utils/managers/preset_manager.py

```diff
@@ -169,8 +169,28 @@ def show_preset(self):
             # save managers parameters in a xml file
             # start = os.path.split(os.path.split(os.path.realpath(__file__))[0])[0]
             # start = os.path.join("..",'daq_scan')
-            ioxml.parameter_to_xml_file(
-                self.preset_params, os.path.join(path, self.preset_params.child('filename').value()))
+            filename_without_extension = self.preset_params.child('filename').value()
+
+            try:
+                ioxml.parameter_to_xml_file(self.preset_params,
+                                            os.path.join(path, filename_without_extension),
+                                            overwrite=False)
+            except FileExistsError as currenterror:
+                logger.warning(str(currenterror)+"File " + filename_without_extension + ".xml exists")
+                confirmation = QtWidgets.QMessageBox()
```

Comment:
I create a method for dealing with that, look at: *pymodaq.utils.messenger.dialog*

Reply:
thanks for the guiding, i thought i should implement the box somewhere else but wasn't sure where... :)